src/main/java/com/prgrms/vouchermanager/repository/VoucherMemoryRepository.java

```diff
@@ -0,0 +1,28 @@
+package com.prgrms.vouchermanager.repository;
+
+import com.prgrms.vouchermanager.domain.Voucher;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Repository;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+@Repository
+@Profile("dev")
+public class VoucherMemoryRepository implements VoucherRepository {
+
+    Map<UUID, Voucher> vouchers = new HashMap<>();
```

Comment:
```suggestion
    private final Map<UUID, Voucher> vouchers = new HashMap<>();
```

인스턴스 필드를 정의 할 때에는 최초에 private final을 붙이는 습관을 기르시는 것을 권장합니다.
이유는 현재 패키지 내에서 접근이 가능하며 Map 자체를 외부에서 초기화하는 것도 가능하여 문제가 발생할 요소가 보이기 때문입니다.